proto/osmosis/superfluid/superfluid.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package osmosis.superfluid;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/x/superfluid/types";
+
+// SuperfluidAsset is a unit to manage superfluid assets.
+message SuperfluidAsset {
+  string denom = 1;
+  // TODO: do we need to keep superfluid_staked_amount in this object?
+  string superfluid_staked_amount = 2 [ 
+    (gogoproto.moretags) = "yaml:\"superfluid_staked_amount\"",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  bool enabled = 3;
+  string risk_adjust_ratio = 4 [
+    (gogoproto.moretags) = "yaml:\"risk_adjust_ratio\"",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+}
```

Comment:
We need two different structs here imo.

We need an enum describing superfluid assets, then a `SuperfluidAssetType` which specifies `denom, enum`.

I think superfluid_staked_amount, enabled, and risk_adjust_ratio shouldn't be in this struct.

`enabled` should be by virtue of being in the enabled superfluid asset list.

Risk adjust ratio will get computed by the state machine somehow. I think we will need to be storing historical records of the risk adjust ratio there, which is why I think it shouldn't go in this struct.
(Also it should be called `risk_adjusted_osmo_equivalent` I think)

Reply:
Modified a little based on your comments here. https://github.com/osmosis-labs/osmosis/pull/432/commits/18ef459a61da4795f22b834a8c5a50ce6cb2a8b0

Reply:
If we are good with current protobuf, I will start implementation of types and keepers.